docs/pages/cdp/cdp-Deploying-your-first-nodejs-app.md

```diff
@@ -33,41 +33,59 @@ CICS TS V5.5 introduced support to run Node.js applications and is required by t
+   create : nodejsapps/myexpressapp.profile
+   create : .zosattributes
+   create : META-INF
+   create : META-INF/cics.xml
+   CICS Bundle generated with bundleid "myexpressapp"</pre>
 
 5. Deploy the CICS bundle into CICS.
 
-    ```console
-    zowe cics-deploy push bundle --name Express --overwrite
-    ```
+   ```text
+   zowe cics-deploy push bundle --name Express --overwrite
+   ```
+
+   A progress bar is shown with status messages as the CICS bundle is deployed and the application is started. This can take a few minutes. The command will summarize the deployment, including the Node.js runtime `.stdout` and `.stderr` file names that the application will write output to:
+
+   <pre class="messageText">
+   Regions in scope 'CICPY000' of CICSplex 'ZOSPTINT':
+   Applid: CICPY000   jobname: CICPY000   jobid: STC35860   sysname: MV2C
 
-    A progress bar is shown with status messages as the CICS bundle is deployed and the application is started. This can take a few minutes. If there are errors, retry with the `--verbose` option for more detailed output, or refer to [Troubleshooting](cdp-Troubleshooting-General).
+   NODEJSAPP resources for bundle 'Express' in scope 'CICPY000':
+   NODEJSAPP resource 'Express' is in 'ENABLED' state in region 'CICPY000' with process id '16844444'.
+   stdout: /u/cicprov/mnt/CICPY000/workdir/CICPY000/myexpressapp/Express/D20190612.T144609.stdout
+   stderr: /u/cicprov/mnt/CICPY000/workdir/CICPY000/myexpressapp/Express/D20190612.T144609.stderr
+   PUSH operation completed</pre>
 
-    This results in a CICS BUNDLE resource named `Express` being defined, installed, and enabled in CICS. If the BUNDLE `Express` was already defined or installed in CICS, it is undeployed first. As the BUNDLE is enabled, the Node.js application is started.
+   This results in a CICS BUNDLE resource named `Express` being defined, installed, and enabled in CICS. If the BUNDLE `Express` was already defined or installed in CICS, it is undeployed first. As the BUNDLE is enabled, the application is started. If there are errors, retry with the `--verbose` option for more detailed output, or refer to [Troubleshooting](cdp-Troubleshooting-General).
 
-6. Test the Node.js application.
+6. Test the application.
 
-    You can call the application from a browser by using URL [http://myzos:3000/](http://myzos:3000/), replacing _myzos_ with the host name of the z/OS system, and _3000_ with the port specified in step 4.
+   You can call the application from a browser by using URL [http://myzos:3000/](http://myzos:3000/), replacing _myzos_ with the host name of the z/OS system, and _3000_ with the port specified in step 4.
 
-    If you make changes to the application, you can redeploy it by repeating step 5.
+   If you make changes to the application, you can redeploy it by repeating step 5.
+
+7. View the application output files.
+
+   Use Zowe CLI commands or the Visual Studio Code extension to view the application output files, for example:
```

Comment:
Suggest clarifying 'the Visual Studio Code extension', perhaps add 'for Zowe' and maybe link.

Reply:
I've added a link and re-ordered the sentence slightly to fit with the example that follows.